### DISCUSSION TITLE
Would you like to try out picows library for websockets?

### DISCUSSION THREAD

**Discussion by tarasko** [Drakkar-Software/OctoBot#2745] [General]
Hi!
Would you like to try out picows for exchange connectivity instead of websockets?

picows was born as a replacement for aiohttp because aiohttp got overwhelmed by trade ticks from binance.
Checkout the project page and benchmarks:

https://github.com/tarasko/picows

**Comment by Herklos** [comment]
Hello @tarasko, can you please send us logs about when aiohttp is overwhelmed by trade ticks from binance?
